data/sql/updates/pending_db_world/rev_1699566747105058500.sql

```diff
@@ -0,0 +1,157 @@
+-- Magistrate Barthilas
+-- move Position
+DELETE FROM `waypoint_data` WHERE `id`=104350;
+INSERT INTO `waypoint_data` (`id`, `point`, `position_x`, `position_y`, `position_z`, `orientation`, `delay`, `move_type`, `action`, `action_chance`, `wpguid`) VALUES (104350, 1, 3696.79, -3605.93, 139.041, NULL, 1000, 1, 0, 100, 0),
+(104350, 2, 3725.58, -3599.48, 142.367, NULL, 1000, 1, 104350, 100, 0);
+
+-- Teleport after moving to the gate
+DELETE FROM `waypoint_scripts` WHERE `guid`=938;
+INSERT INTO `waypoint_scripts` (`id`, `delay`, `command`, `datalong`, `datalong2`, `dataint`, `x`, `y`, `z`, `o`, `guid`) VALUES 
+(104350, 1, 6, 329, 1, 0, 4068.28, -3535.68, 122.771, 2.5, 938);
+
+
+
```

Comment:
Waypoint data based on sniffs from WoW Classic today.

```suggestion
-- Magistrate Barthilas
-- move Position
DELETE FROM `waypoint_data` WHERE `id`=104350;
INSERT INTO `waypoint_data` (`id`, `point`, `position_x`, `position_y`, `position_z`, `orientation`, `delay`, `move_type`, `action`, `action_chance`, `wpguid`) VALUES
(104350, 1, 3707.6914, -3603.0557, 140.96613, NULL, 0, 1, 0, 100, 0),
(104350, 2, 3708.6914, -3602.8057, 141.21613, NULL, 0, 1, 0, 100, 0),
(104350, 3, 3710.4414, -3602.5557, 141.46613, NULL, 0, 1, 0, 100, 0),
(104350, 4, 3713.1914, -3602.3057, 141.46613, NULL, 0, 1, 0, 100, 0),
(104350, 5, 3717.1914, -3601.8057, 141.96613, NULL, 0, 1, 0, 100, 0),
(104350, 6, 3717.4414, -3601.8057, 142.21613, NULL, 0, 1, 0, 100, 0),
(104350, 7, 3717.9414, -3601.8057, 142.21613, NULL, 0, 1, 0, 100, 0),
(104350, 8, 3719.6914, -3601.3057, 142.21613, NULL, 0, 1, 0, 100, 0),
(104350, 9, 3721.9414, -3600.8057, 142.46613, NULL, 0, 1, 0, 100, 0),
(104350, 10, 3723.083, -3600.4836, 142.3591, NULL, 0, 1, 104350, 100, 0);

-- Teleport after moving to the gate
DELETE FROM `waypoint_scripts` WHERE `guid`=938;
INSERT INTO `waypoint_scripts` (`id`, `delay`, `command`, `datalong`, `datalong2`, `dataint`, `x`, `y`, `z`, `o`, `guid`) VALUES
(104350, 1, 6, 329, 1, 0, 4068.74, -3535.97, 122.825, 0.2521562, 938);
```

Reply:
Using these coordinates, the creature moves a little strangely near the door, and it rocks back and forth. The position of the video is 16s. Also, in WLK, pause at the first path location and then continue moving.
https://clipchamp.com/watch/nlqxBPy20JW Creatures shake

Reply:
You are absolutely correct, I noticed that too. But these are the values that the real game is using, so if we want to make it look better we should fix the pathing code to improve the movement. But I think that's outside the scope of your PR - better to put in the waypoints accurately now so that if the movement is fixed in the future it will already be correct.

Here is the sniff section I got that from:

```
ServerToClient: SMSG_ON_MONSTER_MOVE (0x2DD4) Length: 105 ConnIdx: 1 Time: XXX Number: 745
MoverGUID: Full: <REDACTED> Creature/0 R4391/S25255 Map: 329 (Stratholme) Entry: 10435 Low: 5710953
Position: X: 3663.2322 Y: -3619.1384 Z: 137.98363
(MovementMonsterSpline) Id: 278
(MovementMonsterSpline) Destination: X: 0 Y: 0 Z: 0
(MovementMonsterSpline) CrzTeleport: False
(MovementMonsterSpline) StopDistanceTolerance: 0
(MovementMonsterSpline) (MovementSpline) Flags: 2432696320 (Unknown5, Steering, Unknown10)
(MovementMonsterSpline) (MovementSpline) Elapsed: 0
(MovementMonsterSpline) (MovementSpline) MoveTime: 3260
(MovementMonsterSpline) (MovementSpline) FadeObjectTime: 0
(MovementMonsterSpline) (MovementSpline) Mode: 0
(MovementMonsterSpline) (MovementSpline) TransportGUID: Full: 0x0
(MovementMonsterSpline) (MovementSpline) VehicleSeat: -1
(MovementMonsterSpline) (MovementSpline) Face: 0 (None)
(MovementMonsterSpline) (MovementSpline) PointsCount: 1
(MovementMonsterSpline) (MovementSpline) VehicleExitVoluntary: False
(MovementMonsterSpline) (MovementSpline) Interpolate: False
(MovementMonsterSpline) (MovementSpline) PackedDeltasCount: 6
(MovementMonsterSpline) (MovementSpline) HasSplineFilter: False
(MovementMonsterSpline) (MovementSpline) HasSpellEffectExtraData: False
(MovementMonsterSpline) (MovementSpline) HasJumpExtraData: False
(MovementMonsterSpline) (MovementSpline) HasAnimTierTransition: False
(MovementMonsterSpline) (MovementSpline) [0] Points: X: 3684.3516 Y: -3605.4714 Z: 137.8665
(MovementMonsterSpline) (MovementSpline) [0] WayPoints: X: 3663.792 Y: -3618.055 Z: 138.17506
(MovementMonsterSpline) (MovementSpline) [1] WayPoints: X: 3664.292 Y: -3615.805 Z: 137.92506
(MovementMonsterSpline) (MovementSpline) [2] WayPoints: X: 3666.042 Y: -3614.805 Z: 137.92506
(MovementMonsterSpline) (MovementSpline) [3] WayPoints: X: 3668.542 Y: -3613.555 Z: 137.92506
(MovementMonsterSpline) (MovementSpline) [4] WayPoints: X: 3678.042 Y: -3608.805 Z: 137.92506
(MovementMonsterSpline) (MovementSpline) [5] WayPoints: X: 3684.042 Y: -3605.555 Z: 138.17506
(MovementMonsterSpline) (MovementSpline) CalculatedSpeedXY: 7.716592
```

Reply:
**I got all the sniff data**

`ServerToClient: SMSG_ON_MONSTER_MOVE (0x2DD4) Length: 117 ConnIdx: 1 Time: 06/24/2023 06:27:23.693 Number: 838
MoverGUID: Full: 0x204CB029200A30C00007080000161C3F Creature/0 R4908/S1800 Map: 329 Entry: 10435 Low: 1449023
Position: X: 3674.6128 Y: -3610.5562 Z: 137.76337
(MovementMonsterSpline) Id: 291
(MovementMonsterSpline) Destination: X: 0 Y: 0 Z: 0
(MovementMonsterSpline) CrzTeleport: False
(MovementMonsterSpline) StopDistanceTolerance: 0
(MovementMonsterSpline) (MovementSpline) Flags: 2432696320 (Unknown5, Steering, Unknown10)
(MovementMonsterSpline) (MovementSpline) Elapsed: 0
(MovementMonsterSpline) (MovementSpline) MoveTime: 3391
(MovementMonsterSpline) (MovementSpline) FadeObjectTime: 0
(MovementMonsterSpline) (MovementSpline) Mode: 0
(MovementMonsterSpline) (MovementSpline) TransportGUID: Full: 0x0
(MovementMonsterSpline) (MovementSpline) VehicleSeat: -1
(MovementMonsterSpline) (MovementSpline) Face: 0 (None)
(MovementMonsterSpline) (MovementSpline) PointsCount: 1
(MovementMonsterSpline) (MovementSpline) VehicleExitVoluntary: False
(MovementMonsterSpline) (MovementSpline) Interpolate: False
(MovementMonsterSpline) (MovementSpline) PackedDeltasCount: 9
(MovementMonsterSpline) (MovementSpline) HasSplineFilter: False
(MovementMonsterSpline) (MovementSpline) HasSpellEffectExtraData: False
(MovementMonsterSpline) (MovementSpline) HasJumpExtraData: False
(MovementMonsterSpline) (MovementSpline) HasAnimTierTransition: False
(MovementMonsterSpline) (MovementSpline) [0] Points: X: 3700.2996 Y: -3604.1274 Z: 139.57762
(MovementMonsterSpline) (MovementSpline) [0] WayPoints: X: 3678.456 Y: -3608.3418 Z: 138.1705
(MovementMonsterSpline) (MovementSpline) [1] WayPoints: X: 3684.206 Y: -3605.5918 Z: 138.4205
(MovementMonsterSpline) (MovementSpline) [2] WayPoints: X: 3684.706 Y: -3605.5918 Z: 138.1705
(MovementMonsterSpline) (MovementSpline) [3] WayPoints: X: 3685.206 Y: -3605.3418 Z: 138.4205
(MovementMonsterSpline) (MovementSpline) [4] WayPoints: X: 3686.206 Y: -3605.3418 Z: 138.4205
(MovementMonsterSpline) (MovementSpline) [5] WayPoints: X: 3689.706 Y: -3605.0918 Z: 138.6705
(MovementMonsterSpline) (MovementSpline) [6] WayPoints: X: 3689.956 Y: -3605.0918 Z: 138.6705
(MovementMonsterSpline) (MovementSpline) [7] WayPoints: X: 3695.706 Y: -3604.5918 Z: 139.1705
(MovementMonsterSpline) (MovementSpline) [8] WayPoints: X: 3698.706 Y: -3604.3418 Z: 139.4205
(MovementMonsterSpline) (MovementSpline) CalculatedSpeedXY: 7.8269234`

```
(MovementMonsterSpline) (MovementSpline) [0] WayPoints: X: 3678.456 Y: -3608.3418 Z: 138.1705
(MovementMonsterSpline) (MovementSpline) [1] WayPoints: X: 3684.206 Y: -3605.5918 Z: 138.4205
```
**and the above** 

`(MovementMonsterSpline) (MovementSpline) [4] WayPoints: X: 3678.042 Y: -3608.805 Z: 137.92506 Superimposed It should be moved to Position: X: 3674.6128 Y: -3610.5562 Z: 137.76337`

 **So the 4 and 5 in the data you gave Do you still need to keep them?**